### PR TITLE
doc: remove references to unversioned repository addresses

### PR DESCRIPTION
**Pull request by gregsfortytwo**
Signed-off-by: Greg Farnum <gfarnum@redhat.com>

### CODE REVIEW

File: doc/start/quick-start-preflight.rst [hunk 1]
```diff
@@ -63,18 +58,19 @@ For CentOS 7, perform the following steps:
...
      name=Ceph noarch packages
-     baseurl=https://download.ceph.com/rpm/el7/noarch
+     baseurl=https://download.ceph.com/rpm-{ceph-stable-release}/el7/noarch
      enabled=1
      gpgcheck=1
      type=rpm-md
      gpgkey=https://download.ceph.com/keys/release.asc
 
-   and then this *Control-D*.  This will use the latest stable Ceph release. If you would like to install a different release, replace ``https://download.ceph.com/rpm/el7/noarch`` with ``https://download.ceph.com/rpm-{ceph-release}/el7/noarch`` where ``{ceph-release}`` is a release name like ``luminous``.
+   and then type *Control-D*.
```
Comment (alfredodeza) [inline doc/start/quick-start-preflight.rst]: I know this isn't being added on this PR, but I would just love to remove the `cat` and `Ctrl-D` references here. Just saying that `/etc/yum.repos.d/ceph.repo` needs to be edited with the following contents should be good enough. 

We don't use this `cat` and `Ctrl-D` elsewhere

Comment (liewegas) [reply doc/start/quick-start-preflight.rst]: We can still provide something cut and pasteable with
<pre>
cat << EOM > foo
line1
line2
EOM
</pre>

**Comment by liewegas** [comment]
Otherwise the change is good!

**Comment by gregsfortytwo** [comment]
Updated to use a here-document format for cut-and-paste-success!

**Pull request merged**